### DISCUSSION TITLE
Pinyin Type Problem

### DISCUSSION THREAD

**Discussion by Lemonawa** [monkeytypegame/monkeytype#2777] [Ideas]
As a chinese, we type pinyin like "ni hao", not "ní hǎo", so pls fix it.
![image](https://user-images.githubusercontent.com/69106050/160266055-0ff65c3d-4f2e-47da-8bf4-d1afc80ca7ba.png)

**Comment by Miodec** [comment]
Are you saying its a grammatical error or that you just generally don't use the `ǎ` character even if its correct? If that's the case you can try enabling lazy mode.

**Reply by Lemonawa** [reply]
hmm, as a chinese, we never use characters like `ǎ`, we just use `a`

**Reply by nonusx** [reply]
> Are you saying its a grammatical error or that you just generally don't use the `ǎ` character even if its correct? If that's the case you can try enabling lazy mode.

Of course we don't use tone mark like `ǎ`. Few input methods support characters like `ǎ`, and using these is also a very troublesome choice when typing. So given that this repo is a web page for testing typing speed, it shouldn't have any characters like `ǎ`. Please replace them with the corresponding English letters.
